just-install.json

```diff
@@ -398,6 +398,14 @@
       },
       "version": "5.0.4"
     },
+    "lockhunter": {
+      "installer": {
+        "kind": "innosetup",
+        "x86": "http://lockhunter.com/exe/lockhuntersetup_{{.version}}.exe",
+        "x86_64": "http://lockhunter.com/exe/lockhuntersetup_{{.version}}.exe"
```

Comment:
There's no need to provide the x86_64 entry. just-install automatically falls-back to x86 if it's the only one available, even on 64-bit systems. Add the x86_64 entry only if there's an actual 64-bit installer.